frontend/src/components/authentication/CreateAccountPage/CreateAccountPage.tsx

```diff
@@ -118,112 +128,147 @@ const CreateAccountPage = (): JSX.Element => {
             return false;
         }
+        setUserTypeError("");
         return true;
     }
 
-    const validateName = (): boolean => {
+    const validateName = (firstName: string, lastName: string): boolean => {
         /*
         Desc: Validates firstName and lastName
         Return: boolean (true if valid, false if not)
         */
-        if (firstName === "" || lastName === "") {
-            alert("Please add your first and last name");
-            return false;
+        //setErrorMessages({...errorMessagesInitial});
+        if (validateFirstName(firstName) && validateLastName(lastName)) {
+            return true;
+        } else if (!validateFirstName(firstName) && !validateLastName(lastName)) {
+            setNameError("Please enter your full name");
         }
+        return false;
+    }
+    const validateFirstName = (firstName: string): boolean => {
+        /*
+        Desc: Validates firstName and lastName
+        Return: boolean (true if valid, false if not)
+        */
+        //setErrorMessages({...errorMessagesInitial});
+        if (firstName === "") {
+            setNameError("Please enter your first name");
+            return false;
+        } 
+        setNameError("");
+        return true;
+    }
+
+    const validateLastName = (lastName: string): boolean => {
+        /*
+        Desc: Validates firstName and lastName
+        Return: boolean (true if valid, false if not)
+        */
+        //setErrorMessages({...errorMessagesInitial});
+        if (lastName === "") {
+            setNameError("Please enter your last name");
+            return false;
+        } 
+        setNameError("");
         return true;
     }
 
-    const validatePhoneNumber = (): boolean => {
+    const validatePhoneNumber = (phoneNumber: string): boolean => {
         /*
         Desc: Validates phone number
         Return: boolean (true if valid, false if not)
         */
         if (phoneNumber === "") {
-            alert("Please add a phone number");
+            setPhoneNumberError("Please enter a phone number");
             return false;
         }
+        if(!processPhoneNumber(phoneNumber)) {
+            return false;
+        }
+        setPhoneNumberError("");
         return true;
     }
 
-    const validateEmail = (): boolean => {
+    const validateEmail = (email: string): boolean => {
         /*
         Desc: Validates email
         Return: boolean (true if valid, false if not)
         */
         if (email === "") {
-            alert("Please add email");
+            setEmailError("Please enter your email");
             return false;
         }
         else if (!email.includes("@")) {
-            alert("Please enter a valid email address");
+            setEmailError("Please enter a valid email address");
             return false;
         }
         //else if (check if email already exists)
         //alert("Account with this email already exists") 
```

Comment:
From manually checking, this error works so you may delete the comment
<img width="426" alt="Screen Shot 2022-08-09 at 10 34 09 AM" src="https://user-images.githubusercontent.com/42549561/183721667-eda2ddeb-0a6b-4b6b-b8c2-2c972cff947b.png">

```suggestion
```